src/solver/ts-generator/hydro.cpp

```diff
@@ -284,7 +285,7 @@ bool GenerateHydroTimeSeries(Data::Study& study, Solver::IResultWriter& writer)
               [&writer, &progression](const Data::Area& area)
               {
                   const int precision = 0;
-                  std::string mcYear = "mc-" + 0;
+                  std::string mcYear = fmt::format("mc-{}", 0);
```

Comment:
No need for `fmt::format` here
```suggestion
                  const std::string mcYear = "mc-0";
```